native/acter/src/api/room.rs

```diff
@@ -645,7 +640,7 @@ impl Room {
             .await?
     }
 
-    pub async fn set_name(&self, name: Option<String>) -> Result<OwnedEventId> {
+    pub async fn set_name(&self, name: String) -> Result<OwnedEventId> {
```

Comment:
we can't unset the name anymore? That sounds like the API should still allow for it ...

Reply:
`matrix_sdk::room::Room::set_name()` doesn't accept `Option<String>`.